samples/excel/85-preview-apis/worksheet-page-layout.yaml

```diff
@@ -0,0 +1,198 @@
+                    ["E Farms", "Apple", "Conventional", 245, 2200],
+                    ["E Farms", "Kiwi", "Conventional", 200, 1500],
+                    ["F Farms", "Kiwi", "Organic", 100, 150],
+                    ["F Farms", "Lemon", "Conventional", 150, 270],
+                    ["G Farms", "Lime", "Organic", 300, 2000],
+                    ["G Farms", "Lemon", "Organic", 250, 1800],
+                    ["G Farms", "Orange", "Organic", 200, 2200],
+                    ["H Farms", "Lime", "Conventional", 80, 1000],
+                    ["H Farms", "Lemon", "Conventional", 75, 1230],
+                    ["H Farms", "Orange", "Conventional", 25, 800],
+                    ["H Farms", "Orange", "Organic", 20, 500],
+                    ["H Farms", "Lemon", "Organic", 10, 770],
+                    ["H Farms", "Kiwi", "Conventional", 30, 300],
+                    ["I Farms", "Lime", "Organic", 50, 400],
+                    ["I Farms", "Apple", "Organic", 275, 220],
+                    ["I Farms", "Kiwi", "Organic", 200, 120],
+                    ["I Farms", "Apple", "Conventional", 100, 3000],
+                    ["J Farms", "Apple", "Organic", 80, 2800],
+                    ["J Farms", "Lime", "Conventional", 160, 2700],
+                    ["K Farms", "Orange", "Conventional", 180, 2000],
+                    ["K Farms", "Apple", "Conventional", 245, 2200],
+                    ["L Farms", "Kiwi", "Conventional", 200, 1500],
+                    ["L Farms", "Kiwi", "Organic", 100, 150],
+                    ["L Farms", "Lemon", "Conventional", 150, 270]
+                ]);
+
+                farmTable.getRange().format.autofitColumns();
+                printSheet.activate();
+                await context.sync();
+            });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            } catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to change page layout and other settings for printing a worksheet.</p>
+        </section>
+        <section class="setup ms-font-m">
+          <h3>Setup</h3>
+          <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Create sample data</span>
+            </button>
+        </section>
+        <section class="setup ms-font-m">
+            <h3>Try it out</h3>
+            <p>In Excel, choose <b>View > Page Layout</b>, then observe the page layout changes as you press the following buttons.</p>
+            <button id="setPageBreaks" class="ms-Button">
+                <span class="ms-Button-label">Add page break</span>
+            </button><p/>
+            <button id="center" class="ms-Button">
```

Comment:
Still seeing weird behavior when I click this button in the sequence it's presented. In **View > Page Layout** mode, when I click the "Center data" button immediately after clicking the "Add page break" button, the first page of data appears to be centered vertically (eventually, after the screen refreshes) but the second page does not appear to be centered vertically. However, if I choose **File > Print** I can see in the print preview that both pages of data are centered. Seems like this flaky behavior could be avoided if you move this button down to the next section of the task pane (before the "Set to black and white printing" button)?